src/rail/pipelines/degradation/apply_phot_errors.py

```diff
@@ -25,11 +25,13 @@
     lsst = dict(
         ErrorModel='LSSTErrorModel',
         Module='rail.creation.degraders.photometric_errors',
+        Bands=['u', 'g', 'r', 'i', 'z', 'y'],
     ),
-    roman = dict(
-        ErrorModel='RomanErrorModel',
-        Module='rail.creation.degraders.photometric_errors',
-    ),
+    #roman = dict(
+    #    ErrorModel='RomanErrorModel',
+    #    Module='rail.creation.degraders.photometric_errors',
+    #    Bands=['F', 'H', 'J', 'Y'],
```

Comment:
it's slightly confusing that Rubin bands are in ascending WL order and Roman bands are descending WL/alphabetical, maybe switch to ['Y', 'J', 'H', 'F']?

Reply:
Makes sense, done.